docs/kyma/docs/031-gs-local-installation.md

```diff
@@ -15,35 +15,38 @@ To run Kyma locally, clone this Git repository to your local machine and downloa
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.10.0
 - [Helm](https://github.com/kubernetes/helm) 2.8.2
 - [jq](https://stedolan.github.io/jq/)
-- [Hyperkit driver](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#hyperkit-driver) - Mac only
-- [Virtualbox](https://www.virtualbox.org/) - Linux or Windows
+
+Vitrualization:
+
+- [Hyperkit driver](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#hyperkit-driver) - Mac
+- [Virtualbox](https://www.virtualbox.org/) - Linux, Windows, Mac
 - Hyper-V - Windows
 
-Read the [prerequisite reasoning](019-prereq-reasoning.md) document to learn why Kyma uses these tools.
+> **NOTE:** To work with Kyma, use only the provided installation and deinstallation scripts. Kyma does not work on a basic Minikube cluster that you can start using the `minikube start` command or stop with the `minikube stop` command. If you do not need Kyma on Minikube anymore, remove the cluster with the `minikube delete` command.
 
 ## Setup certificates
```

Comment:
doesn't seem like you took into account comment from Andreas about installing certificate on linux